src/model/extension.ts

```diff
@@ -166,8 +206,16 @@ export async function initializeExtension(
         updateStateCallback,
       );
       break;
-
-    default:
+    
+    case PathStyleExtension.extensionType:
+      extensionModel = new PathStyleExtension(
+        model,
+        layerModel,
+        updateStateCallback,
+      );
+      break;
+    
+        default:
```

Comment:
This should be dedented (that should also help fix CI). You can run `yarn install && yarn run fmt` from the root of the directory to run code formatting on the javascript code.